lib/holidays/use_case/context/dates_driver_builder.rb

```diff
@@ -15,6 +15,8 @@ def call(start_date, end_date)
           clean(dates_driver)
         end
 
+        #FIXME Why is the date_driver set to optional as the first param? That's
+        # just plain wrong...
         def build(dates_driver = {}, date)
```

Comment:
Wohhhh, i have mistaken again.Sorry.First params is not optional, so i have checked that in L-21.